go/x509/verify.go

```diff
@@ -145,12 +146,14 @@ const (
 
 // isValid performs validity checks on the c.
 func (c *Certificate) isValid(certType int, currentChain []*Certificate, opts *VerifyOptions) error {
-	now := opts.CurrentTime
-	if now.IsZero() {
-		now = time.Now()
-	}
-	if now.Before(c.NotBefore) || now.After(c.NotAfter) {
-		return CertificateInvalidError{c, Expired}
+	if !opts.DisableTimeChecks {
+		now := opts.CurrentTime
+		if now.IsZero() {
+			now = time.Now()
```

Comment:
Nit: Seems that time.Now() returns the current local, are the NotBefore/NotAfter values in the certificate converted to local time? (otherwise they'd be UTCTime, https://tools.ietf.org/html/rfc5280#section-4.1.2.5)